### DISCUSSION TITLE
Configuration on quarkus leads to non correct ingress setup - Security issue with the latest image.

### DISCUSSION THREAD

**Discussion by cirix** [keycloak/keycloak#19597] [Feedback]
Hi all,
I have been building my own keycloak image based on 21.0.2 and of course I use builder container pattern. 
The above docker is getting successfully build in my  repos and pushed to registry without any problem.
Logs from ingress controller:
![image](https://user-images.githubusercontent.com/924122/230683313-4c4dff12-83ff-4113-a8e0-c498e3b69c80.png)

On top,
the image build has some critical CVE's

![image](https://user-images.githubusercontent.com/924122/230683474-7f3dfc75-54ec-431c-b2cd-7c868d2c41ce.png)

Do you plan addressing the two above important issues. Specially the one with security is critical and the other, looks like actually that quarkus starts on only  8443 as such the old setup that was sending traffic from 8080->8443 has not been ported.

**Comment by abstractj** [comment]
@cirix it seems that you asked two questions, and I only have an answer for one :)

CVEs fixes are planned by milestones and will be available in the upcoming releases.  For more information, please see our list of issues: https://github.com/keycloak/keycloak. If the CVE reported by the security scanner is not available on the list of our issues, it is recommended to create a new public GH issue. Dependencies with unpatched or unfixed vulnerabilities should be excluded from the report before submitting them.

Our team recently updated most of its dependency and this should be reflected by scanning the nightly images, and they should be part of upcoming releases:
```
trivy image --ignore-unfixed --timeout 25m quay.io/keycloak/keycloak:nightly
```
